tests/client_test.py

```diff
@@ -16,14 +18,14 @@ class TestClient(object):
 
     @classmethod
     def setup_class(cls):
-        with open('json/credentials.json') as cred:
+        with open(resource_filename('json/credentials.json')) as cred:
```

Comment:
These are unit tests for the client class with mocked responses (via tests/json) so they don't need real credentials. They use the filler values to instantiate a client object but I suppose we could mock the `Oauth1Authenticator` object.